### PR TITLE
Winner/partial generated plot logging

### PR DESCRIPTION
**Pull request by aleksai**
Add in harvester logs message on proofs like:
```
2021-06-28T15:54:10.550 harvester chia.harvester.harvester: DEBUG    ✅ Proofs were found by: {'plot-k25-2021-06-25-15-37-ec359f6147c72473c756c8b2f766baefaf23393b54a16a97be5e65c14f3ffcd0.plot (non-portable)'}
2021-06-28T15:54:52.618 harvester chia.harvester.harvester: DEBUG    ✅ Proofs were found by: {'plot-k25-2021-06-25-21-08-39c1a3e01df90f53b41fd88309b70ba8fa0ec4302cf82eb8fef5d2a198efafa0.plot (P2: txch1gg8wgwqkgf55nmt03nafzmjnv939tr437358zwpuzhv4f03rygss9yzppn)'}
```

I'm proposing this useful info imo on DEBUG level of logging, but if it can be considered as INFO, can change it.

### CODE REVIEW

**Review by mariano54 — APPROVED** [review]
LGTM (After fixing lint)! any rationale why put it in DEBUG and not INFO? Is it to avoid breaking current tooling which reads log file?

**Comment by alek-sai** [comment]
@mariano54 you are right, must be info. Debug isn't having any sense.

**Comment by alek-sai** [comment]
I didn't understand first part of your sentence btw, I'm trying to find those lint mistakes I possibly made, but can't found any related to code I've added. I'm not so familiar with LGTM, can you point me if possible?

**Comment by alek-sai** [comment]
Thank you for collaboration and lgtm acronym clarifying, Gene! I've checked lgtm.com and seems like no issues https://lgtm.com/projects/g/Chia-Network/chia-blockchain/rev/pr-367609de987007fc6e03b7bd431af9f521fca680

**Comment by alek-sai** [comment]
@mariano54 @hoffmang9 guys, should I still do something or to relax and wait? happy phase 2 btw :)

File: chia/harvester/harvester_api.py [hunk 1]
```diff
@@ -67,6 +70,9 @@ async def new_signage_point_harvester(
             # This means that we have not received the handshake yet
             return None
 
+        config = load_config(DEFAULT_ROOT_PATH, "config.yaml")
```
Comment (mariano54) [inline chia/harvester/harvester_api.py:73]: We don't want to load the config on every run. Use the harvester config: `self.harvester.config`, or even better just print out the puzzle hash. The harvester knows nothing about address encodings

**Review by mariano54 — CHANGES_REQUESTED** [review]
Also, i think you need to start again from the main branch, looks like some tests are failing

**Pull request closed without merging**